pkg/operation/botanist/controlplane.go

```diff
@@ -1158,17 +1126,68 @@ func (b *Botanist) DeployETCD(ctx context.Context) error {
 			foundStatefulset = false
 		}
 
+		if foundEtcd && etcd.Status.Etcd.Name != "" {
+			name = etcd.Status.Etcd.Name
+		}
+
+		defragmentSchedule, err := DetermineDefragmentSchedule(b.Shoot.Info, etcd)
+		if err != nil {
+			return err
+		}
+		etcdValues["defragmentSchedule"] = defragmentSchedule
+
+		switch role {
+		case common.EtcdRoleMain:
+			etcdValues["metrics"] = "extensive" // etcd-main emits extensive (histogram) metrics
+
+			hvpaValues["enabled"] = hvpaEnabled
+			hvpaValues["minAllowed"] = map[string]interface{}{
+				"cpu":    "200m",
+				"memory": "700M",
+			}
+			hvpaValues["maintenanceWindow"] = b.Shoot.Info.Spec.Maintenance.TimeWindow
```

Comment:
Move out of `switch`  block. Or have we decided to allow HVPA update etcd-events irrespective of maintenance window?